nautobot/core/templates/inc/nav_menu.html

```diff
@@ -178,7 +220,20 @@
 
         // Save the scroll position when the navbar is scrolled
         navbar.addEventListener('scroll', debounce(function() {
-            localStorage.setItem('navbarScrollPosition', navbar.scrollTop);
+            sessionStorage.setItem('navbarScrollPosition', navbar.scrollTop);
         }, 250));
+
+         // Add click event listeners to dropdown links
+        addLinkClickListeners();
+
+        // Apply the 'active' class to the previously clicked link
+        if (activeLink) {
+            var previouslyClickedLink = document.querySelector('.navbar-fixed-left .navbar-nav > .dropdown > .nav-dropdown-menu > li > a[href="' + activeLink + '"]');
+            var currentLocation = window.location.pathname + window.location.search;
+            
+            if (previouslyClickedLink && currentLocation.includes(previouslyClickedLink.getAttribute('href'))) {
+                previouslyClickedLink.parentElement.classList.add('active');
+            }
```

Comment:
Should we clear the `activeLink` session storage in an `else` case here? Just wondering if this gives rise to a situation where the link goes from active to inactive back to active as the user navigates around the app between various pages.

Reply:
Updated. Wasn't sure if it mattered too much, but it'll now clear the `activeLink` session storage if navigating away from the initial page